plugins/modules/monitoring/uptimerobot.py

```diff
@@ -8,138 +8,180 @@
 
 
 DOCUMENTATION = '''
-
 module: uptimerobot
-short_description: Pause and start Uptime Robot monitoring
+short_description: Create, remove, pause and start monitors in Uptime Robot
 description:
-    - This module will let you start and pause Uptime Robot Monitoring
-author: "Nate Kingsley (@nate-kingsley)"
+    - This module lets you create, remove, pause and start monitors in Uptime Robot.
+author:
+- "Nate Kingsley (@nate-kingsley)"
+- "Ninjiner (@Ninjiner)"
 requirements:
     - Valid Uptime Robot API Key
 options:
     state:
         description:
             - Define whether or not the monitor should be running or paused.
+            - C(created), C(absent) and C(present) were added in community.general 1.3.0.
```

Comment:
I would ditch the first sentence and rephrase the second one like "Define the state of the monitor" and explain each state, for the version this is redundant with the field `version_added`.